indexing-service/src/main/java/org/apache/druid/indexing/overlord/MetadataTaskStorage.java

```diff
@@ -208,15 +208,15 @@ public void setStatus(final TaskStatus status)
   }
 
   @Override
-  public List<TaskInfo<Task, TaskStatus>> getRecentlyFinishedTaskInfo(
+  public List<TaskInfo<Task, TaskStatus>> getFinishedTaskInfoByCreatedTimeDuration(
```

Comment:
Yes. `getRecentlyCreatedFinishedTaskInfo()` is more obvious than `getFinishedTaskInfoByCreatedTimeDuration()` and still contain relation to `createdTime` parameter.
Maybe it will be better add delimiter between `Created` and `Finished` like `getRecentlyCreatedAlreadyFinishedTaskInfo`.

Reply:
Thanks, sounds good to me.